integrations/zio-prelude/src/main/scala/sttp/tapir/codec/zio/prelude/newtype/TapirNewtypeSupport.scala

```diff
@@ -0,0 +1,17 @@
+package sttp.tapir.codec.zio.prelude.newtype
+
+import sttp.tapir._
+import zio.prelude.Newtype
+
+trait TapirNewtypeSupport[A] { self: Newtype[A] =>
```

Comment:
would it be possible to also define the codec/schema externally, that is without modifying the newtype itself?

Reply:
Sure, I'll try to implement it.